### ISSUE TITLE
Missing required keys: :query_hash?

### ISSUE THREAD

**Issue by thiagotonon** [ankane/pghero#131]
![](https://user-images.githubusercontent.com/5451970/29382943-ac6ba78a-82a4-11e7-939a-beaf8c2d4476.png)

When i try to enter `/queries` url, an `500 error` shows up and Rails gives me this:

```
ActionView::Template::Error: No route matches {:action=>"show_query", :controller=>"pg_hero/home", :database=>"primary", :query_hash=>nil} missing required keys: [:query_hash]
```

Running `SELECT * FROM pg_stat_statements;` shows me that some items has a nil `queryid` column. What am I doing wrong in this case?

**Comment by ankane** [comment]
Hey @thiagotonon, thanks for the report 👍 Should be fixed once #127 is merged.

**Comment by thiagotonon** [comment]
@ankane thanks for the reply! I've made #132 to fix this issue, really need this merged asap.
